lms/djangoapps/survey/models.py

```diff
@@ -261,3 +259,14 @@ def save_answers(cls, form, user, answers, course_key):
                 answer.field_value = value
                 answer.course_key = course_key
                 answer.save()
+
+    @classmethod
+    def retire_user(cls, user_id):
+        answers = cls.objects.filter(user=user_id)
+        if not answers:
+            return False
+
+        for answer in answers:
```

Comment:
This is fine.  You could also call `.update(field_value='')` on answers to do all the updates in a single query.

Reply:
@iloveagent57 Took in your feedback. You wanna review again?